### PR TITLE
super important sum() util

### PR DESCRIPTION
**Pull request by tdumitrescu**
implemented without reduce and other trickery, to keep it lightweight and fast enough

### CODE REVIEW

**Comment by jordanmnunez** [comment]
lgtm 👍

**Comment by ilyakamens** [comment]
LGTM

File: lib/util/index.js [hunk 1]
```diff
@@ -153,6 +153,14 @@ export function pluralize(singular, number, plural) {
...
 }
 
+export function sum(arr) {
+  let _sum = 0;
+  for (let i = 0; i < arr.length; i++) {
+    _sum += arr[i];
+  }
+  return _sum;
+}
+
```
Comment (evnp) [inline lib/util/index.js]: Why not the traditional `arr.reduce((a, b) => a + b, 0)`? Either way is cool, just curious

Comment (evnp) [inline lib/util/index.js]: jesus, need to read comments. Ignore me :D

Comment (tdumitrescu) [inline lib/util/index.js]: haha well another point only in the tests, not github description, is that this implementation works out of the box for non-array objects like `arguments`

Comment (evnp) [inline lib/util/index.js]: Ah cool, the more you know. As soon as you put reduce and prototype bs together the above starts to look pretty nice in terms of readability

**Pull request merged**